providers/dns/dns_providers.go

```diff
@@ -212,6 +212,8 @@ func NewDNSChallengeProviderByName(name string) (challenge.Provider, error) {
 		return netlify.NewDNSProvider()
 	case "nifcloud":
 		return nifcloud.NewDNSProvider()
+	case "njalla":
+		return namedotcom.NewDNSProvider()
```

Comment:
Got it, this seems to be it. Did you forget to replace that?

Reply:
Fixed